### ISSUE TITLE
SVG image in dataURL is incorrectly minified and breaks further minification

### ISSUE THREAD

**Issue by phiw13** [jakubpawlowicz/clean-css#583]
clean-css v 3.2.10
running: cleancss -b -d -o style.min.css style.css

input:
```
.class { background: url("data:image/svg+xml,<svg version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' width='16px' height='16px' viewBox='0 0 16 16' xml:space='preserve'><style type='text/css'><![CDATA[.st0{fill:none;stroke:hsl(220,5%,35%');stroke-width:2;stroke-linejoin:round;stroke-miterlimit:10;stroke-linecap:round}
]]></style><polyline class='st0' points='2,12 8,4 14,12 '/></svg>") 50% / 16px no-repeat hsl(45,95%,70%);
	border-radius: .28571428571429em ;
	display: inline-block;
	width: 2em;
	height: 2em;
}

@media (min-width:668px) {
	.other-class {
		-webkit-flex: 1 1 auto;
		flex: 1 1 auto;
		overflow: hidden;
		text-overflow: ellipsis;
		padding-right: 1em;
	}
/* and way more stuff */
```

output:

```
.class {background:url('data:image/svg+xml,<svg version='1.1' xmlns='http://www.w3.org/2000/svg' xmlns:xlink='http://www.w3.org/1999/xlink' x='0px' y='0px' width='16px' height='16px' viewBox='0 0 16 16' xml:space='preserve'><style type='text/css'><![CDATA[.st0{fill:none;stroke:hsl(220,5%,35%');stroke-width:2;stroke-linejoin:round;stroke-miterlimit:10;stroke-linecap:round}
]]></style><polyline class='st0' points='2,12 8,4 14,12 '/></svg>') 50% / 16px no-repeat hsl(45,95%,70%);
	border-radius: .28571428571429em ;
	display: inline-block;
	width: 2em;
	height: 2em;
}

@media (min-width:668px) {
	.other-class {
		-webkit-flex: 1 1 auto;
		flex: 1 1 auto;
		overflow: hidden;
		text-overflow: ellipsis;
		padding-right: 1em;
	}
/* and nothing minified further down the sheet */

Note the hierachy of quotes in the dataURL: double `"` around the URL, single `'` around the attribute values inside the SVG image (In my experience, browsers are unhappy and confused unless this is coded this way).

(Omitting the `-b` flag doesn’t change anything)

Ofcourse, using a properly base64 encoded dataURL works correctly…

**Comment by jakubpawlowicz** [comment]
Good catch - I think we should be smarter about choosing the right kind of quotes.

**Comment by jakubpawlowicz** [comment]
@phiw13 Fixed in master - feel free to use 3.3-pre, `--skip-rebase` option (if possible) or wait until it's backported to 3.2.11.
